include/sbat.h

```diff
@@ -6,6 +6,9 @@
 #ifndef SBAT_H_
 #define SBAT_H_
 
+#define SBAT_INIT "sbat,1,2021030218\n"
```

Comment:
What would be the policy to update this? I wonder if we should have a `sbat_version.h.in` or something with a `@@SBAT_DATE@@` that's replaced with the build date ?

Reply:
Maybe SBAT_VAR_SIG instead of SBAT_INIT?

Reply:
Certainly not an automatically generated date.
and yes to SBAT_VAR_SIG